src/time/span.cr

```diff
@@ -334,87 +366,105 @@ struct Time::Span
     end
   end
 
-  def self.zero
+  def self.zero : Time::Span
     ZERO
   end
 
-  def zero?
+  def zero? : Bool
     to_i == 0 && nanoseconds == 0
   end
 end
 
 struct Int
-  def week
+  # :nodoc:
```

Comment:
Why?

Reply:
That was exactly what I thought when I saw this abomination.
"We don't want aliases, oh except for when they make a kewl DSL even cooler."
If someone has been conditioned to like this, they can keep using it, but new people don't have to know this.

It's a big amount of noise in the docs, so I did not want to document it. But I also didn't want to not document it.

Reply:
If you all agree on removing these aliases it's fine with me. Maybe it's not that bad to write `1.minutes` instead of `1.minute`.

Reply:
Either have them and document it, or not have them. I'd vote for the latter.